src/client/entity-editor/relationship-editor/relationship-editor.tsx

```diff
@@ -289,9 +333,15 @@ class RelationshipModal
 	}
 
 	renderEntitySelect() {
-		const {baseEntity, relationshipTypes} = this.props;
+		const {baseEntity, relationshipTypes, seriesType} = this.props;
 		const {targetEntity} = this.state;
-		const types = getValidOtherEntityTypes(relationshipTypes, baseEntity);
+		let types;
+		if (baseEntity.type === 'Series') {
+			types = [seriesType];
+		}
+		else {
+			types = getValidOtherEntityTypes(relationshipTypes, baseEntity);
+		}
```

Comment:
Are you sure this is needed? Doesn't `getValidOtherEntityTypes` return the one valid relationship type (X is part of Series)?

Reply:
**getValidOtherEntityTypes** will return all the valid relationship type for series entity i.e `["Author", "Edition", "EditionGroup", "Publisher", "Work"]` 
This will then show all entities in relationship search results
Example: 
![Screenshot from 2021-06-23 15-03-06](https://user-images.githubusercontent.com/55311336/123073611-2fc8aa80-d434-11eb-9a29-d85f20163eae.png)

But we don't want this. Instead we want to restrict the search results to just one entity type. For Example If the user select series type as 'Author' then show only authors in the search results:
Example: 
![Screenshot from 2021-06-23 15-06-21](https://user-images.githubusercontent.com/55311336/123074052-9bab1300-d434-11eb-9e72-39a2358180a9.png)
 Hence I think it is necessary to use seriesType value (from the SeriesSection) here.

Reply:
Ah, yes, of course there is a relationship type "part of series" for each entity type, I forgot about that.

That makes sense. However I'm worried that we won't be able to have any other type of relationship for series.
For example imagine we want to add a new relationship type "Series was inspired by Work" (quick example: This sci-fi series  by Dan Simmons was inspired by Homer's Illiad: https://en.wikipedia.org/wiki/Ilium/Olympos).
With the current setup we won't be able to search for a work or add a relationship to it.

It's not something we need to solve in this PR, but let's keep in mind that we'll probably have to change that.
The UI changes I have in mind might solve the issue by having a separate component in the SeriesSection where we can search for and add entities to the series, while the relationship editor would be used for the other types of relationships. But let's cross that bridge when we get to it :)

Reply:
sounds great. Ready to Get it :100: :100: